lib/coordinate.rb

```diff
@@ -0,0 +1,28 @@
+class Coordinate 
+  attr_accessor :x, :y 
+
+  def initialize(x, y)
+    @x = x
+    @y = y
+  end
+
+  def up
+    @y += 1
+    self
```

Comment:
Same point here Coordinates are like number - they're value objects. `up`, `down` etc. may want to return new coordinates.